css/src/components/checkbox.scss

```diff
@@ -0,0 +1,110 @@
+/* stylelint-disable selector-max-specificity, max-nesting-depth, selector-no-qualifying-type */
+
+$checkbox-border-radius: $radius-sm !default;
+$checkbox-unchecked-border-color: $text !default;
+$checkbox-color: $primary !default;
+$checkbox-check-color: $primary-invert !default;
+$checkbox-check-hover-color: $text-subtle !default;
+$checkbox-size: 1.25em !default;
+$checkbox-icon-size: 0.75em !default;
+$checkbox-timing-function: $input-timing-function !default;
+$checkbox-duration: $input-transition-duration !default;
+$checkbox-spacing: $spacer-3;
+
+@mixin checkmark {
+	display: block;
+	position: absolute;
+	top: 0.1875em;
+	width: 0.3125em;
```

Comment:
Shouldn't these two values `top` and `width` be variables? As it's done for height. It feels like if height can be changed, these two should be flexible as well since they're tight together.

Reply:
Added variables for top, width, and border-width.